### ISSUE TITLE
Question: mentions and commenting?

### ISSUE THREAD

**Issue by SoniEx2** [friendica/friendica#5146]
https://github.com/kensanata/bitlbee-mastodon/issues/25#issuecomment-392758288

Does friendica support commenting without notification? A "comment" in my dictionary is an associated post (i.e. reply) without any mentions. It links back to the original post and is linked from the original post, but doesn't annoy the OP.

**Comment by annando** [comment]
We are sending salmon notifications and are doing the appropriate things in the entry content. We don't send a visible ```@username```, since this shouldn't be needed from a technical perspective.

**Comment by MrPetovan** [comment]
The short answer is no. Like @annando said, we are sending the proper protocol messages when a comment is posted, actual notifications are up to the user on their system.

**Comment by SoniEx2** [comment]
That's so BS, thanks. Please add a way to manage invisible mentions.

**Comment by MrPetovan** [comment]
No. It is practically impossible to enforce outside of Friendica, runs counter to the federated protocols Friendica is based on, you can achieve the same result by quoting the OP in a post of your own, and it isn’t nicely asked for.

**Comment by tobiasd** [comment]
What is a toot? Sorry to ask but never heard about it before in Freindica. And follow-up, what is a *LB toot*

**Comment by tobiasd** [comment]
Thanks! And LB toots?

**Comment by tobiasd** [comment]
So when I get you correct, then you want a native reshare ability? Then this would be a duplicate of feature request #838

**Comment by MrPetovan** [comment]
I don't understand what LB toots are, but a quick search gives 
https://github.com/tootsuite/mastodon/issues/7080, https://github.com/tootsuite/mastodon/issues/7111 and https://github.com/nolanlawson/pinafore/issues/207 

It seems to be a creation by @SoniEx2, and by no mean something that "has been known for a while", even less accepted.

It isn't the first time @SoniEx2 has been rude or inconsiderate as well, so I really am not interested in finding out why we should spend any time looking into this.

**Comment by SoniEx2** [comment]
mastodon doesn't have text search so I can't exactly search for posts starting with "LB". I do see them all the time tho.

**Comment by MrPetovan** [comment]
How is it different from a quote-tweet in Twitter or a shared post in Friendica?

**Comment by SoniEx2** [comment]
Well uh, it is *kinda* like a quote-tweet. But it also involves a boost.

The only thing is that you can't find quote-tweets from the quoted tweet, IIRC. (e.g. find a quote-tweet, click the quote, and try to look for the quote-tweet from that page. you shouldn't be able to find it.) That's what should be different - you should be able to easily go back and forth between the two posts.

**Comment by SoniEx2** [comment]
Here's an example public "LB toot", found randomly going through my timeline: https://scholar.social/@theartguy/100086272792716062

And this is the post it refers to: https://elekk.xyz/@noelle/100086254089147865 (you have to look it up manually, because that's one of the issues with "LB toots")

Note that there's no easy way to find one from the other. Finding the "LB toot" is basically impossible, and finding the original post is also difficult (but significantly easier).

**Comment by MrPetovan** [comment]
This sounds like the pingback feature of yore. We would send an automatic reply to the quoted post with a back reference to the quoting post.

I assume this would only apply to quoted posts (share + text) as native reshares should still collect replies on the original post. this doesn't happen on Friendica yet (see #838) but that's what should happen.

**Comment by MrPetovan** [comment]
Let's take the example:
- UserA posts PostA
- UserB posts PostB that quotes PostA along with some text

Currently, UserA isn't notified their post has been quoted, and there's no way from PostA to find PostB.

What you're proposing is:
- UserA posts PostA
- UserB posts PostB that quotes PostA along with some text
- An automated reply is sent under PostA with a link to PostB

This way, UserA is notified of the existence of PostB (but not of the subsequent replies on PostB) and there's a link from PostA to PostB via the automated reply.

**Comment by SoniEx2** [comment]
No, UserA shouldn't get a notification (which is why I'm asking for the option to strip mentions off replies).

**Comment by SoniEx2** [comment]
uh I don't think you're quite understanding the nuances of mastodon and mastodon culture...

I'm not qualified to explain this any further, sorry.

**Comment by MrPetovan** [comment]
No problem.
